cli/bpmetadata/types.go

```diff
@@ -68,11 +75,54 @@ type BlueprintDescription struct {
 	Architecture []string `json:"architecture,omitempty" yaml:"architecture,omitempty"`
 }
 
+type BlueprintDeploymentTime struct {
+	Configuration int `json:"configuration,omitempty" yaml:"configuration,omitempty"`
+	Deployment    int `json:"deployment,omitempty" yaml:"deployment,omitempty"`
+}
+
+type BlueprintCostEstimate struct {
+	Description string `json:",omitempty" yaml:",omitempty"`
+	Url         string `json:",omitempty" yaml:",omitempty"`
+}
+
+type BlueprintCloudProduct struct {
+	ProductId   string
+	PageUrl     string
+	Label       string
+	LocationKey bool
+}
+
+type BlueprintQuotaDetail struct {
+	QuotaType            string              `yaml:"quotaType"`
```

Comment:
This reads like an enum

Reply:
It's `Fixed` or `Dynamic`. I've added a enum to restrict the values.